src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -1162,20 +1177,15 @@ export default {
 		}
 
 		blockquote {
-			position: relative;
+			padding-left: 0;
```

Comment:
You don't need it I think, please check.

Reply:
for rtl padding-inline-start will be converted to padding-right, so you'll have them on both sides